.github/workflows/rocky_makelang.yml

```diff
@@ -36,9 +36,14 @@ jobs:
       - name: Install Octopoes
         run: pip install wheel && cd octopoes && python setup.py bdist_wheel && pip install dist/octopoes*.whl
 
-      - name: Generate the .po files if they changed
+      - name: Generate the .pot file if source strings changed
         run: SECRET_KEY="whatever" make lang
         working-directory: ./rocky
 
-      - name: Check if there are any changed files
-        run: git diff --exit-code
+      - name: Check if at least one source string changed
+        run: |
+          changed_lines=$(git diff --unified=0 rocky/rocky/locale/django.pot | grep -o '^[-+]' | wc -l)
```

Comment:
This is a bit hard to understand, you could perhaps also grep against strings that match `msgid` or `msgstr`?

Reply:
I've updated the regex, should be more explicit now